client-src/elements/chromedash-userlist.ts

```diff
@@ -1,17 +1,13 @@
+import {SlCheckbox} from '@shoelace-style/shoelace';
 import {LitElement, css, html, nothing} from 'lit';
+import {customElement, state} from 'lit/decorators.js';
 import {SHARED_STYLES} from '../css/shared-css.js';
+import {User} from '../js-src/cs-client.js';
 
+@customElement('chromedash-userlist')
 class ChromedashUserlist extends LitElement {
-  static get properties() {
-    return {
-      users: {attribute: false},
-    };
-  }
-
-  constructor() {
-    super();
-    this.users = [];
-  }
+  @state()
+  users!: User[];
```

Comment:
This needs to be initialized for the uses inside this file to work:
```suggestion
  users: User[] = [];
```